src/components/cluster/detail/add_node_pool.js

```diff
@@ -0,0 +1,220 @@
+                // onChange={this.updateAWSInstanceType}
+                readOnly={false}
+                // value={this.state.aws.instanceType.value}
+                value='m3.large'
+              />
+              <p>2 CPU cores, 8GB RAM each</p>
+            </FlexWrapperDiv>
+          </label>
+          <label className='availability-zones' htmlFor='availability-zones'>
+            <span className='label-span'>Availability Zones</span>
+            <FlexWrapperDiv>
+              <NumberPicker
+                // label=''
+                max={this.props.maxAvailabilityZones}
+                min={this.props.minAvailabilityZones}
+                // onChange={this.updateAvailabilityZonesPicker}
+                readOnly={false}
+                stepSize={1}
+                // value={this.state.availabilityZonesPicker.value}
+                value={3}
+              />
+              <p>
+                or <a href='#'>Select distinct availability zones</a>
+              </p>
+            </FlexWrapperDiv>
+            <p>
+              Covering one availability zone, the worker nodes of this node pool
+              will be placed in the same availability zones as the
+              cluster&apos;s master node.
+            </p>
+          </label>
+          <label className='scaling-range' htmlFor='scaling-range'>
+            <span className='label-span'>Scaling range</span>
+            <NodeCountSelector
+              // autoscalingEnabled={this.isScalingAutomatic(
+              //   this.props.provider,
+              //   this.state.releaseVersion
+              // )}
+              autoscalingEnabled={true}
+              label={{ max: 'MAX', min: 'MIN' }}
+              // onChange={this.updateScaling}
+              readOnly={false}
+              // scaling={this.state.scaling}
+              scaling={{
+                automatic: false,
+                min: 3,
+                minValid: true,
+                max: 3,
+                maxValid: true,
+              }}
+            />
+          </label>
+          <FlexWrapperDiv>
+            <Button
+              bsSize='large'
+              bsStyle='primary'
+              // disabled={!this.valid()}
+              // loading={this.state.submitting}
+              // onClick={this.createCluster}
+              type='button'
+            >
+              Create Node Pool
+            </Button>
+            <Button
+              bsSize='large'
+              bsStyle='default'
+              // disabled={!this.valid()}
+              // loading={this.state.submitting}
+              // onClick={this.createCluster}
+              style={{ background: 'red' }}
+              type='button'
+            >
+              Cancel
+            </Button>
+          </FlexWrapperDiv>
+          <p style={{ fontSize: '16px' }}>
+            Note that it takes around 15 minutes on average until a new node
+            pool is fully available.
+          </p>
```

Comment:
FYI: Initially we wont have that data yet, so we should remove the entire paragraph `Note that it takes around`...

I'm wondering about the `<p style={{ fontSize: '16px' }}>`. Why is that necessary? 16px should be the standard font size.

Reply:
Because I'm overwriting it in the other `<p>`s